src/RollbarLogHandler.php

```diff
@@ -46,98 +40,102 @@ class RollbarLogHandler extends AbstractLogger
+     * @param string $level
+     *
+     * @throws \InvalidArgumentException
      */
-    public function __construct(RollbarNotifier $rollbar, Application $app, $level = 'debug')
+    public function __construct(Application $app, $level = 'debug')
     {
-        $this->rollbar = $rollbar;
-
         $this->app = $app;
-
+        
         $this->level = $this->parseLevel($level ?: 'debug');
     }
-
+    
+    /**
+     * Parse the string level into a Monolog constant.
+     *
+     * @param  string $level
+     *
+     * @return int
+     *
+     * @throws \InvalidArgumentException
+     */
+    protected function parseLevel($level)
+    {
+        if (isset($this->levels[$level])) {
+            return $this->levels[$level];
+        }
+        
+        throw new InvalidArgumentException('Invalid log level: '.$level);
+    }
+    
     /**
      * Log a message to Rollbar.
      *
-     * @param mixed  $level
+     * @param mixed $level
      * @param string $message
-     * @param array  $context
+     * @param array $context
+     *
+     * @throws \InvalidArgumentException
      */
     public function log($level, $message, array $context = [])
     {
         // Check if we want to log this message.
         if ($this->parseLevel($level) < $this->level) {
             return;
         }
-
+        
         $context = $this->addContext($context);
-
+        
         if ($message instanceof Exception) {
-            $this->rollbar->report_exception($message, null, $context);
+            Rollbar::log(Level::error(), $message, $context)->getUuid();
         } else {
-            $this->rollbar->report_message($message, $level, $context);
+            $level = $level ? Level::fromName($level) : Level::error();
+            Rollbar::log($level, $message, $context)->getUuid();
         }
     }
-
+    
     /**
      * Add Laravel specific information to the context.
      *
      * @param array $context
+     *
+     * @return array
      */
     protected function addContext(array $context = [])
     {
         // Add session data.
         if ($session = $this->app->session->all()) {
-            if (empty($this->rollbar->person) or ! is_array($this->rollbar->person)) {
-                $this->rollbar->person = [];
-            }
-
+            $person = [];
+            
             // Merge person context.
             if (isset($context['person']) and is_array($context['person'])) {
-                $this->rollbar->person = $context['person'];
+                $person = $context['person'];
                 unset($context['person']);
-            } else {
-                if ($this->rollbar->person_fn && is_callable($this->rollbar->person_fn)) {
-                    $data = @call_user_func($this->rollbar->person_fn);
-                    if (isset($data['id'])) {
-                        $this->rollbar->person = call_user_func($this->rollbar->person_fn);
-                    }
-                }
```

Comment:
Why remove support for `person_fn`?  One of the recent issues in this repo led to people specifically using the `person_fn` functionality.